addons/website/static/src/scss/website.scss

```diff
@@ -1422,24 +1422,38 @@ $-o-carousel-controllers-size: var(--CarouselControllers-size, #{map-get($spacer
 
 // Parallax
 .parallax {
-    // TODO this introduces a limitation: no dropdown will be able to
-    // overflow. Maybe there is a better way to find.
-    &:not(.s_parallax_no_overflow_hidden) {
+    // Kept for compatibility. Before "s_parallax_bg" was wrapped by an
+    // intermediate parent element to clip it ("s_parallax_bg_wrap"), its direct
+    // parent was the section with the "parallax" class, which had the CSS rule
+    // "overflow: hidden". That was not the right solution because it introduced
+    // a limitation: no dropdown could overflow, for example.
+    &:not(.s_parallax_no_overflow_hidden):not(:has(> .s_parallax_bg_wrap)) {
         overflow: hidden;
     }
 
-    > .s_parallax_bg {
+    > .s_parallax_bg, > .s_parallax_bg_wrap > .s_parallax_bg {
         @extend %o-we-background-layer;
+
+        &.o_bg_img_opt_repeat {
+            background-repeat: repeat;
+        }
     }
     @include media-breakpoint-up(xl) {
         // Fixed backgrounds are disabled when using a mobile/tablet device,
         // which is not a big deal but, on some of them (iOS...), defining the
         // background as fixed breaks the background-size/position props.
         // So we enable this only for >= XL devices.
-        &.s_parallax_is_fixed > .s_parallax_bg {
-            background-attachment: fixed;
+        &.s_parallax_is_fixed {
+            > .s_parallax_bg, > .s_parallax_bg_wrap > .s_parallax_bg {
+                background-attachment: fixed;
+            }
         }
     }
+
+    > .s_parallax_bg_wrap {
+        @include o-position-absolute(0, 0, 0, 0);
```

Comment:
Useless, no? Is not that automatic for background layers? :thinking:

Reply:
> Useless, no? Is not that automatic for background layers? 🤔

👍🏼 I removed it and added the `> .s_parallax_bg_wrap` selector above where `%o-we-background-layer` is already used for `.s_parallax_bg`.